packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf

```diff
@@ -154,8 +155,8 @@ dbms.connector.https.address=localhost:#{default.https.port}
 # must be the configured IP address for one of the local interfaces.
 #ha.host.data=127.0.0.1:6001
 
-# The interval at which slaves will pull updates from the master. Comment out
-# the option to disable periodic pulling of updates. Unit is seconds.
+# The interval, in seconds, at which slaves will pull updates from the master. You must comment out
+# the option to disable periodic pulling of updates.
 ha.pull_interval=10
```

Comment:
Fine but by default HA mode is disabled: https://github.com/maxsumrall/neo4j/blob/f4fc002ef31ad14d16a78e0beb4a09cccf37a906/packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf#L137